File: Programing_Basics_with_JS/examPreparation/programmingBasicsOnlineExam6And7April2019/01.OscarsCeremony.js
function oscarsCeremony(input) {
    let roomRent = Number(input[0]);

    let statuettesPrice = roomRent * 0.7;
    let cateringPrice = statuettesPrice * 0.85;
    let soundSystemPrice = cateringPrice * 0.5;

    let totalCost = roomRent + statuettesPrice + cateringPrice + soundSystemPrice;

    console.log(totalCost.toFixed(2));
}

oscarsCeremony(['3500'])
oscarsCeremony(['5555'])

File: Programming_Basics_with_Python/programming_basics_online_exam_6_and_7_april_2019/01._oscars_ceremony.py
def oscars_ceremony(hall_rent, statuettes_price, catering_price, voiceover):
    total_expenses = hall_rent + statuettes_price + catering_price + voiceover

    return total_expenses


hall_rent = int(input())
statuettes_price = hall_rent * 0.7
catering_price = statuettes_price * 0.85
voiceover = catering_price / 2

print(f'{oscars_ceremony(hall_rent, statuettes_price, catering_price, voiceover):.2f}')
